mep/accounts/models.py

```diff
@@ -398,48 +398,46 @@ class DatePrecision(Flags):
 
 class DatePrecisionField(models.PositiveSmallIntegerField):
```

Comment:
Needs documentation; looks like we should also add a `description` based on the django docs - https://docs.djangoproject.com/en/2.0/howto/custom-model-fields/#documenting-your-custom-field